sysutils/screen/Portfile

```diff
@@ -73,7 +79,7 @@ platform darwin {
     }
 }
 
-platform darwin 8 {
+platform darwin < 10 {
```

Comment:
Sorry, I was replying from mobile earlier, it was difficult to type special characters.
It should be:
```
if {${os.platform} eq "darwin" && ${os.major} < 10} {
```

Reply:
No worries, I suspected that I'd done it wrong. Fixed in latest commit